bundles/org.eclipse.passage.lbc.jetty/src/org/eclipse/passage/lbc/internal/jetty/i18n/JettyMessages.properties

```diff
@@ -11,9 +11,10 @@
 #     ArSysOp - initial API and implementation
 ###############################################################################
 
-JettyServerLauncher_e_start=Jetty start: Error
-JettyServerLauncher_e_start_exists=Jetty start: already exists
-JettyServerLauncher_e_stop=Jetty stop: Error
-JettyServerLauncher_e_stop_not_started=Jetty stop: not started
-JettyServerLauncher_ok_start=Jetty start: OK
-JettyServerLauncher_ok_stop=Jetty stop: OK
+JettyServerLauncher_server_start_error=[ERROR] Launch Jetty Server with error: "{0}".
+JettyServerLauncher_server_running_error=[ERROR] Jetty Server already running.
```

Comment:
the prefix `[ERROR]` should be removed from the message itself as typically it is added by the logger